examples/examples/submit_and_watch.rs

```diff
@@ -70,7 +70,7 @@ async fn simple_transfer() -> Result<(), Box<dyn std::error::Error>> {
         balance_transfer.find_first_event::<polkadot::balances::events::Transfer>()?;
 
     if let Some(event) = transfer_event {
-        println!("Balance transfer success: value: {:?}", event.2);
+        println!("Balance transfer success: value: {:?}", event);
```

Comment:
```suggestion
        println!("Balance transfer success. Value: {:?}", event);
```
…but is it a "value" or an "event"?

Reply:
Mmm, it was a value but if we are printing out the entire event now by the looks of it, it's now just the Transfer event.